angular.json

```diff
@@ -63,7 +63,7 @@
               ],
               "index": {
                 "input": "src/web/index.prod.html",
-                "output": "index.html",
+                "output": "index.html"
```

Comment:
Was this accidentally committed?

Reply:
seems to be from some part of the setup... it should be this way I think?
Btw there's a CLI thing at the bottom. What would that be?